### DISCUSSION TITLE
Welcome to gourdo1's qmk_firmware Discussions!

### DISCUSSION THREAD

**Discussion by gourdo1** [gourdo1/qmk_firmware#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by liyongg** [comment]
I am new to QMK Firmware flashing and found your excellent mapping, but would like to change a few mappings on the base layer. What would be the correct way to do so?

**Reply by gourdo1** [reply]
@liyongg If those changes aren't already covered by the customization interface built into my firmware, you would have to fork my work and  edit the keymap.c to make the changes you want to make. Then use a utility like QMK_MSYS to compile and flash your firmware to your board.

**Comment by crunzaty** [comment]
I was wondering if you could help me with this one thing i had. It's from an old version of QMK and it really well, but i wanted to make some changes and found out that it doesn't work anymore.

I had my RGB lights for my sides change color when caps lock was on. But the way it's done doesn't work anymore.
My code was:
```
#ifdef RGB_MATRIX_ENABLE

	void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
		
		if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK)) {
			RGB_MATRIX_INDICATOR_SET_COLOR(68, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(69, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(71, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(72, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(74, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(75, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(77, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(78, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(81, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(82, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(84, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(85, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(88, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(89, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(92, 255, 255, 255); //capslock key
			RGB_MATRIX_INDICATOR_SET_COLOR(93, 255, 255, 255); //capslock key
	}
}
#endif // RGB_MATRIX_ENABLE
```

I've tried a bunch of stuff but no matter what i get errors when i try to compile it :(
help would be really appreciated!
